control/mateqn.py

```diff
@@ -69,7 +70,9 @@ def lyap(A,Q,C=None,E=None):
         :math:`A X E^T + E X A^T + Q = 0`
 
     where Q is a symmetric matrix and A, Q and E are square matrices
-    of the same dimension. """
+    of the same dimension.
+
```

Comment:
Not super-familiar with PEP8 and so on, but this extra blank line seems odd.

Reply:
The extra blank line is a `numpydoc` standard: https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard